src/module.ts

```diff
@@ -99,7 +99,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (isNuxt3()) return
 
     // @ts-expect-error TODO: use @nuxt/bridge-schema
-    nuxt.hook('generate:done', async () => {
+    nuxt.hook('build:compiled', async () => {
```

Comment:
This was correct for generating a site. We need to add an additional hook for building, which probably would be copying the lib files into `static`, which isn't ideal 🤔

```suggestion
    nuxt.hook('generate:done', async () => {
```